runtime/Makefile

```diff
@@ -21,7 +21,7 @@ GOSUM := $(GOMOD:.mod=.sum)
 DOCKER_IMAGE_TAG?=latest
 GO_CACHE_VOLUME_NAME?=gocache
 FIRECRACKER_CONTAINERD_TEST_IMAGE?=localhost/firecracker-containerd-test
-FICD_DM_POOL?=fc-test-thinpool
+FICD_DM_POOL?=testpool
```

Comment:
The quickstart and getting started guides use `fc-dev-thinpool`. We should probably update those to be consistent.

Reply:
Wait, we have three names...